exercises/two-fer/two_fer_test.rb

```diff
@@ -0,0 +1,41 @@
+require 'minitest/autorun'
+require_relative 'two_fer'
+
+class TwoFerTest < Minitest::Test
+  def  test_empty
+    #skip
+    assert_equal "One for you, one for me.", TwoFer.two_fer
+  end
+
+  def test_alice
+    skip
```

Comment:
I think none of these tests should be skipped? as they are good tests. what do you think @Insti?

Reply:
The `skip`s should be there, they are part of the standard test cases and are added by the test generator. The skip just means that that test won't be run for now, but should be enabled later. This is so the student is only presented with one failing test to make pass at a time.


See: http://exercism.io/languages/ruby/tests
And: https://github.com/exercism/ruby/issues/753